dev-tools/mage/integtest_docker.go

```diff
@@ -95,7 +95,7 @@ func (d *DockerIntegrationTester) Test(_ string, mageTarget string, env map[stri
 	}
 	dockerRepoRoot := filepath.Join("/go/src", repo.CanonicalRootImportPath)
 	dockerGoCache := filepath.Join(dockerRepoRoot, "build/docker-gocache")
-	magePath := filepath.Join("/go/src", repo.CanonicalRootImportPath, repo.SubDir, "build/mage-linux-amd64")
+	magePath := filepath.Join("/go/src", repo.CanonicalRootImportPath, repo.SubDir, "build/mage-linux-"+GOARCH)
```

Comment:
Tried to figure out where `GOARCH` is set. Should this be `runtime.GOARCH`?

Reply:
I see a few examples across the codebase that seem to rely on `GOARCH` being visible without any scoping, like https://github.com/elastic/beats/blob/master/dev-tools/mage/build.go#L231, so I followed the example here as well.

Reply:
Got it, makes sense.